queries/dailyVolumeByDate_test.go

```diff
@@ -47,7 +48,18 @@ func TestDailyVolumeByDate_QueryRow(t *testing.T) {
 		wantTomorrowBase          float64
 		wantTomorrowQuote         float64
 	}{
+		// TODO add case for buy base/quote and add trade data in test below accordingly
```

Comment:
please add this case in a subsequent PR (better to separate out so we're not complicating this PR)

Reply:
That was the plan - the TODO was so I'd remember this in the next step of implementation. Should I not do that?